### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by damccorm**

### CODE REVIEW

**Review by Oriblish — COMMENTED** [review]
gh pr checkout 2
[Licensed](https://github.com/actions/checkout/actions/workflows/licensed.yml)

**Comment by Oriblish** [comment]
🧩

> gh pr checkout 2 [Licensed](https://github.com/actions/checkout/actions/workflows/licensed.yml)
gh pr checkout 2

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+# Usage
+
+See [action.yml](action.yml)
+
+Basic:
+```yaml
+actions:
+- uses: actions/checkout@latest
+- uses: actions/setup-node@latest
```
Comment (haaami01) [inline README.md]: ref

Comment (Almaani41) [reply README.md]: ### 
[Uploading Lecture16.pdf…]()

**Review by yydslaomaque — COMMENTED** [review]
1

**Review by qxMaTR1xp — APPROVED** [review]

**Review by qxMaTR1xp — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+This action checks out your repository so that your workflow operates from the root of the repository
+
+# Usage
+
+See [action.yml](action.yml)
+
+Basic:
+```yaml
+actions:
+- uses: actions/checkout@latest
```
[the remark this answers is not in the document]
Comment (qxMaTR1xp) [reply README.md]: @@@@

[the remark this answers is not in the document]
Comment (qxMaTR1xp) [reply README.md]: > make sure the ref exists. :)

**Comment by qxMaTR1xp** [comment]
@@qxMaTR1xp

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (qxMaTR1xp) [reply README.md]: microsoft-contribution-license-agreement.pdf

@@@README.txt

[the remark this answers is not in the document]
Comment (qxMaTR1xp) [reply README.md]: > make sure the ref exists. :)README.txt

**Review by qxMaTR1xp — COMMENTED** [review]
@qxMaTR1xp

**Review by 113602611780178409309 — COMMENTED** [review]
LICENSE

**Comment by 113602611780178409309** [comment]
https://github.com/actions/checkout/pull/2/commits/6b71935c437479e5c49db23356f45cff7c8532e9

**Pull request merged**